bedrock/mozorg/views.py

```diff
@@ -422,7 +422,7 @@ def new_home(request, locale=None):
 
 def home(request):
```

Comment:
Yes, I agree that we should consolidate to one view and update the tests as necessary.